libnavigation-ui/src/test/java/com/mapbox/navigation/ui/summary/SummaryModelTest.kt

```diff
@@ -18,6 +18,7 @@ import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 import org.robolectric.annotation.Config
 import java.util.Calendar
+import kotlin.jvm.Throws
```

Comment:
NIT: I don't checkout this branch to verify. But is it unused import?

Reply:
It's used down here https://github.com/mapbox/mapbox-navigation-android/blob/17312de630ea31bea4518853e29c6d560755cd8b/libnavigation-ui/src/test/java/com/mapbox/navigation/ui/summary/SummaryModelTest.kt#L104